### PR TITLE
Add CODEOWNERS

### PR DESCRIPTION
**Pull request by martinwicke**
Added what we know about contrib mainly, and some well-separated components.

The list of names behind each component is not perfect, I took this from a partially definitely outdated doc, but I suggest we maintain it here instead of in that doc.

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+tensorflow/contrib/batching/* @alextp @chrisolston
+tensorflow/contrib/bayesflow/* @ebrevdo @rsepassi @jvdillon
+tensorflow/contrib/cmake/* @mrry @benoitsteiner
+tensorflow/contrib/copy_graph/* @tucker @poxvoculi
+tensorflow/contrib/crf/* @kentonl
+tensorflow/contrib/data/* @mrry
+tensorflow/contrib/distributions/* @jvdillon @langmore @rsepassi
+tensorflow/contrib/factorization/* @agarwal-ashish @xavigonzalvo
+tensorflow/contrib/ffmpeg/* @fredbertsch
+tensorflow/contrib/framework/*	joel-shor
```
Comment (jhseu) [inline CODEOWNERS]: `@joel-shor`?

File: CODEOWNERS [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+tensorflow/contrib/graph_editor/*	@purpledog
```
Comment (jhseu) [inline CODEOWNERS]: nit: space instead of tab?

File: CODEOWNERS [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+tensorflow/contrib/imperative/* @keveman
+tensorflow/contrib/integrate/* @shoyer
+tensorflow/contrib/kernel_methods/* @petrosmol
+tensorflow/contrib/ios_examples/* @petewarden
+tensorflow/contrib/labeled_tensor/* @shoyer
+tensorflow/contrib/layers/* @fchollet @martinwicke
+tensorflow/contrib/learn/* @martinwicke @ispirmustafa @alextp
+tensorflow/contrib/linalg/* @langmore
+tensorflow/contrib/linear_optimizer/* @petrosmol @andreasst @katsiapis
+tensorflow/contrib/lookup/*	@ysuematsu @andreasst
```
Comment (jhseu) [inline CODEOWNERS]: space instead of tab?

**Review by jhseu — APPROVED** [review]

File: CODEOWNERS (hunk 1 above)
Comment (martinwicke) [inline CODEOWNERS]: I just copied the file we had. Do you have suggestions?

**Comment by aselle** [comment]
Awesome!

**Pull request merged**